decentraland/unity-renderer
# feat: add FOV slider

## What does this PR change?

Fix #74 
Adds an FOV slider to the quality settings

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/fov-slider-setting
2. Change the FOV slider value in the quality settings
3. Verify that it affects the actual FOV of the camera
4. Close the page and reopen it to verify if it was saved correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
